README.md

```diff
@@ -159,18 +161,12 @@ Running simulations asynchronously allows users to launch multiple simulations i
 ```python
```

Comment:
Should this code snippet contain the vehicle download as well? We don't know if people will use it as is. Moreover, let's do velocity in velocity_list and just set velocity_list=[1, 10, 20, 30, 40]. Otherwise, people will clutter the machines with this snippet.

Reply:
This one works as well, and is just 5 simulations, not 50.